iree/compiler/Dialect/Flow/Analysis/Dispatchability.cpp

```diff
@@ -154,5 +154,10 @@ bool Dispatchability::isDispatchable(FuncOp funcOp) {
   return isDispatchable(funcOp.getName());
 }
 
+bool Dispatchability::isInvalidated(
+    const AnalysisManager::PreservedAnalyses &pa) {
+  return !pa.isPreserved<Dispatchability>();
```

Comment:
I suspect this may be the issue. I think here you want to always return false so that the dispatchability analysis is never invalidated (as the way this analysis works it only ever runs once and the results are always valid).

https://source.cloud.google.com/results/invocations/1b31aef2-b111-4924-a3ce-ab6eca58ede6/targets/iree%2Fgcp_ubuntu%2Fbazel%2Fpresubmit/log

Reply:
fixed